### DISCUSSION TITLE
HA-WyzeAPI 0.1.28b1

### DISCUSSION THREAD

**Discussion by SecKatie** [SecKatie/ha-wyzeapi#637] [General]
## What's Changed
* docs(SECURITY.md): fixes the email address to my latest email by @SecKatie in https://github.com/SecKatie/ha-wyzeapi/pull/599
* docs(CHANGELOG.md): removes the changelog as it has not been maintain… by @SecKatie in https://github.com/SecKatie/ha-wyzeapi/pull/600
* build(poetry): updates homeassistant and fixes python version range to meet new requirements by @SecKatie in https://github.com/SecKatie/ha-wyzeapi/pull/601
* Delete .github/workflows/codeql-analysis.yml by @SecKatie in https://github.com/SecKatie/ha-wyzeapi/pull/602
* Update load and unload platform by @brg468 in https://github.com/SecKatie/ha-wyzeapi/pull/609
* Added battery sensors for WCOv2, BCP, and VDBP cameras by @KyGuy2002 in https://github.com/SecKatie/ha-wyzeapi/pull/606
* Update stale bot by @brg468 in https://github.com/SecKatie/ha-wyzeapi/pull/625
* Support Indicator Floodlight V2 by @jdeath in https://github.com/SecKatie/ha-wyzeapi/pull/616
* Fixed and added entity toggles by @KyGuy2002 in https://github.com/SecKatie/ha-wyzeapi/pull/607
* Update error handling to use HomeAssistantError by @brg468 in https://github.com/SecKatie/ha-wyzeapi/pull/627
* Update Config Flow by @brg468 in https://github.com/SecKatie/ha-wyzeapi/pull/636

## New Contributors
* @KyGuy2002 made their first contribution in https://github.com/SecKatie/ha-wyzeapi/pull/606

**Full Changelog**: https://github.com/SecKatie/ha-wyzeapi/compare/0.1.27...0.1.28b1

<hr /><em>This discussion was created from the release <a href='https://github.com/SecKatie/ha-wyzeapi/releases/tag/0.1.28b1'>HA-WyzeAPI 0.1.28b1</a>.</em>

**Comment by brg468** [comment]
So far the beta looks good to me! I don't have a lot to test from what @KyGuy2002 fixed/added but everything seems to be working fine with my devices.

**Reply by SecKatie** [reply]
That's fantastic! It was a lot of changes so I wanted to ensure that we didn't get a flood of broken installs 😅

**Comment by Happy-Cadaver** [comment]
I only have Wyze light bulbs but everything seems to be fine on 2024.12.0b2
